ext/js/language/ja/japanese.js

```diff
@@ -523,16 +524,39 @@ export function convertHiraganaToKatakana(text) {
  * @param {string} text
  * @returns {string}
  */
-export function convertNumericToFullWidth(text) {
+export function convertAlphanumericToFullWidth(text) {
     let result = '';
```

Comment:
Would this be more performant using an array than string concatenation? Not sure if this is the performance bottleneck